src/Aspire.Dashboard/Model/ResourceIconHelpers.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Dashboard.Model;
+
+using Microsoft.FluentUI.AspNetCore.Components;
+using Icons = Microsoft.FluentUI.AspNetCore.Components.Icons;
+
+internal static class ResourceIconHelpers
+{
+    /// <summary>
+    /// Maps a resource to a default icon.
+    /// </summary>
+    public static Icon GetIconForResource(ResourceViewModel resource)
+    {
+        return resource.ResourceType switch
+        {
+            KnownResourceTypes.Executable => new Icons.Filled.Size24.Database(),
+            KnownResourceTypes.Project => new Icons.Filled.Size24.CodeCircle(),
+            KnownResourceTypes.Container => new Icons.Filled.Size24.Box(),
+            string t => t.Contains("database", StringComparison.OrdinalIgnoreCase) ? new Icons.Filled.Size24.Database() : new Icons.Filled.Size24.SettingsCogMultiple(),
```

Comment:
All databases contain "database" in the type? no "db"?

Reply:
Would be nice to be able to have an attribute or annotation on the resource type to set the icon from the apphost....

Reply:
Yeah, I agree with this. @glennc are you comfortable making that change, or would you like me to?

Reply:
I viewed that as a different PR after this one, because it would require protocol changes. Happy to hand this PR over and do it all in one go if you want though? The logic as it is now copied from the graph. but that would obviously also update if we did the custom icons now.

Reply:
If you handed it off, release would be delayed until 9.3. Just want to set expectatiosn

Reply:
For now I think this is fine